core/src/error.rs

```diff
@@ -54,12 +55,20 @@ pub enum CoreError {
     },
     #[error("Transform does not terminate")]
     NonTerminatingTransform,
-    #[error("Parsing error: {0:#?}.")]
+    #[error("Parsing error: {0}.")]
     Parsing(#[from] ParseError),
     #[error("Native call error: Non-module given to package {0} named {1}")]
     NonModuleToNative(String, String),
     #[error("Root element is not a parent, cannot remove __document for playground")]
     RootElementNotParent,
+    #[error("DenyAllResolver is used; resolving of packages disallowed")]
```

Comment:
I think we should use a variant name and a error message that is more friendly to end-users and does not expose the internals of the system. 
```suggestion
    #[error("Importing external packages is not allowed.")]
```

Reply:
Might be smart, I changed it to your suggestion!